repository.tf

```diff
@@ -114,12 +148,30 @@ resource "azuredevops_git_repository_file" "network" {
 resource "azuredevops_git_repository_file" "virtual_machine" {
   count         = var.create_virtual_machine_template == true ? 1 : 0
   repository_id = azuredevops_git_repository.landing_zone.id
-  file          = "virtual_machine_template.tf"
-  content = templatefile("${path.module}/templates/virtual_machine.tftpl", {
-    stage = var.stage
+  file          = "vm.tf"
+  content = templatefile("${path.module}/templates/vm.tftpl", {
+    create_vm_win = var.create_vm_win
+    create_vm_ux  = var.create_vm_ux
   })
   branch              = "refs/heads/${azuredevops_git_repository_branch.init.name}"
-  commit_message      = "Add virtual_machine_template.tf"
+  commit_message      = "Add vm template files"
+  overwrite_on_create = true
+
+  lifecycle {
+    ignore_changes = [commit_message]
+  }
+}
+
+resource "azuredevops_git_repository_file" "virtual_machine_creation" {
+  count         = var.create_virtual_machine_template == true ? 1 : 0
+  repository_id = azuredevops_git_repository.landing_zone.id
+  file          = "vm_creation.tf"
+  content = templatefile("${path.module}/templates/vm_creation.tftpl", {
+    create_vm_win = var.create_vm_win
+    create_vm_ux  = var.create_vm_ux
+  })
+  branch              = "refs/heads/${azuredevops_git_repository_branch.init.name}"
+  commit_message      = "Add vm module template files"

```

Comment:
Could we merge these 2 files? What is the goal behind having the module and module variables in 2 different files when they are mostly 1:1 the same?

Reply:
Yes, merged in one ,"vm.tf".